huggingface_sb3/push_to_hub.py

```diff
@@ -41,56 +358,63 @@ def _copy_file(filepath: Path, dst_directory: Path):
     shutil.copy(str(filepath.name), str(dst))
 
 
-def push_to_hub(repo_id: str,
-               filename: str,
-               commit_message: str,
-               use_auth_token=True,
-               local_repo_path="hub"):
+def push_to_hub(
+    repo_id: str,
+    filename: str,
+    commit_message: str,
+    token: Optional[str] = None,
+    local_repo_path="hub",
+):
     """
-      Upload a model to Hugging Face Hub.
-      :param repo_id: repo_id: id of the model repository from the Hugging Face Hub
-      :param filename: name of the model zip or mp4 file from the repository
-      :param commit_message: commit message
-      :param use_auth_token
-      :param local_repo_path: local repository path
-      """
-    huggingface_token = HfFolder.get_token()
+    Upload a model to Hugging Face Hub.
+    :param repo_id: repo_id: id of the model repository from the Hugging Face Hub
+    :param filename: name of the model zip or mp4 file from the repository
+    :param commit_message: commit message
+    :param use_auth_token
+    :param local_repo_path: local repository path
+    """
+    if token is None:
+        token = HfFolder.get_token()
 
-    temp = repo_id.split('/')
+    if token is None:
+        raise ValueError(
+            "You must login to the Hugging Face Hub. There are two options: "
+            "(1) Type `huggingface-cli login` in your terminal and enter your token. "
+            "(2) Enter your token in the `token` argument. "
+            "Your token is available in the Settings of your Hugging Face account. "
+        )
+
```

Comment:
I think this can be removed. The downstream functions do a similar check to this.